utils/measure/measure.py

```diff
@@ -162,13 +162,84 @@
     _VERSION = f.read().strip()
 
 class Measure:
-    def __init__(self, light_controller: LightController, power_meter: PowerMeter):
+    """Measure the power usage of a light.
+
+    This class is responsible for measuring the power usage of a light. It uses a LightController to control the light, and a PowerMeter
+    to measure the power usage. The measurements are exported as CSV files in export/<model_id>/<color_mode>.csv (or .csv.gz). The
+    model_id is retrieved from the LightController and color mode can be selected by user input or config file (.env). The CSV files
+    contain one row per variation, where each column represents one property of that variation (e.g., brightness, hue, saturation). The last
+    column contains the measured power value in watt.
+    If you want to generate model JSON files for the LUT model, you can do so by answering yes to the question "Do you want to generate
+    model.json?".
+
+    Example
+    -------
+    
+    >>> from light_controller import LightController
+    >>> from power_meter import PowerMeter
+
+    >>> light_controller = LightController()
+    >>> power_meter = PowerMeter()
+
+    >>> measure = Measure(light_controller, power_meter)
+    >>> measure.start()
+
+    # CSV file export/<model-id>/hs.csv will be created with measurements for HS
+    color mode (e.g., hue and saturation). The last column contains the measured
+    power value in watt.
+
+    # If you answered yes to generating a model JSON file, a model.json will be
+    created in export/<model-id>
+    
+    """
+
+    def __init__(self, light_controller: LightController, power_meter: PowerMeter) -> None:
+        """This class measures the power consumption of the light bulb.
+
+        Parameters
+        ----------
+        light_controller : LightController
+            The light controller to use.
+        power_meter : PowerMeter
+            The power meter to use.
+        """
         self.light_controller = light_controller
         self.power_meter = power_meter
         self.num_0_readings: int = 0
 
-    def start(self):
-        """Starts the measurement session"""
+    def start(self) -> None:
+        """Starts the measurement session.
+
+        This method asks the user for the required information, sets up the light controller and power meter and starts the measurement
+        session.
+
+        Parameters
```

Comment:
This parameter section and returns can be removed.

Reply:
Fixed in [a1dc069](https://github.com/bramstroker/homeassistant-powercalc/pull/951/commits/a1dc069b0bc1f44314f22bd5ed41ad4ef742fee8)